Header header
---
uint8 LEFT=1
uint8 FORWARD=2
uint8 RIGHT=3
uint8 STOP=4
uint8 BACKWARD=5
uint8 dir
